client/capstone-app/src/components/MapWithRoute.jsx

```diff
@@ -2,21 +2,40 @@ import React from 'react';
 import { MapContainer, TileLayer, GeoJSON, Marker } from 'react-leaflet';
 import L from 'leaflet';
 import 'leaflet/dist/leaflet.css';
+import { useMap } from 'react-leaflet'
+import { useEffect } from 'react';
+import { getRandomColor } from '../../utils';
 
-const MapWithRoute = ({ routeData, currentRadius, destinationRadius, centerCoordinates}) => {
+
+
+const MapWithRoute = ({ routeData, directions, centerCoordinates}) => {
   const onEachFeature = (feature, layer) => {
     if (feature.properties && feature.properties.summary) {
       layer.bindPopup(`Distance: ${feature.properties.summary.distance} meters<br>Duration: ${feature.properties.summary.duration} seconds`);
     }
   };
 
   return (
-    <MapContainer center={centerCoordinates} zoom={15} style={{ height: '80vh', width: '100vh' }}>
+    <MapContainer center={[-90.071533, 29.951065]} zoom={15} style={{ height: '60vh', width: '100vh' }}>
```

Comment:
Is there a reason why you reverted to hardcoded coords?

Reply:
It was for testing purposes; it has been changed back in future PRs